en/includes/guides/organization-management/invite-parent-organization-users.md

```diff
@@ -63,3 +63,67 @@ Inorder to view or delete the invitations via {{ product_name }} Console:
     ![View invitations]({{base_path}}/assets/img/guides/organization/manage-organizations/view-invitations.png){: width="600" style="display: block; margin: 0;"}
 
 4. If required, invitations can be deleted by clicking the **Delete** icon.
+
+## Invite users without email notifications
+
+In order to disable email notifications, `manageNotificationsInternally` property can be set to false 
```

Comment:
```suggestion
In order to disable sending email notifications internally from {{ product_name }} , `manageNotificationsInternally` property can be set to false 
```

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4832/commits/a315e362207f14c6a51e0697f231c03ec8aca209